javascript/builtins/PluralRules.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "OverconstrainedError": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PluralRules",
+        "support": {
+          "chrome": {
+            "version_added": "63"
+          },
+          "chrome_android": {
+            "version_added": "63"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "resolvedOptions": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PluralRules/resolvedOptions",
+          "support": {
+            "chrome": {
+              "version_added": null
```

Comment:
For `chrome` and `chrome_android`, the subfeatures don't have data, but webview_android does. Is that intended?